dts/bindings/regulator/x-powers,axp2101-regulator.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright (c), 2021 NXP
+# Copyright (c), 2024 Lothar Felten <lothar.felten@gmail.com>
+# SPDX -License-Identifier: Apache-2.0
+
+description: |
+  AXP2101 PMIC
+
+  The PMIC has five DCDC converters and nine LDOs.
+  All need to be defined as children nodes.
+  For example:
+
+  i2c {
+    pmic@68 {
+      reg = <0x68>;
+      ...
+      regulators {
```

Comment:
you are missing a top-level compatible here, this is a MFD device. It's ok to just add support for regulators now, but DT layout needs to be designed according to how device is.